moviepy/video/VideoClip.py

```diff
@@ -577,8 +546,9 @@ def blit_on(self, picture, t):
         # is the position relative (given in % of the clip's size) ?
         if self.relative_pos:
             for i, dim in enumerate([wf, hf]):
-                if not isinstance(pos[i], str):
-                    pos[i] = dim * pos[i]
+                if isinstance(pos[i], str):
```

Comment:
I'm not sure that this is a worthwhile change.

Reply:
Oh yes true, thanks for catching that.

Reply:
ok, reverted